amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasLikeOperationParser.scala

```diff
@@ -114,12 +108,12 @@ abstract class OasOperationParser(entry: YMapEntry, adopt: Operation => Operatio
           .filter(y => !isOasAnnotation(y.key.as[YScalar].text))
           .foreach {
             entry =>
-              val node = ScalarNode(entry.key).text()
+              val node = ScalarNode(entry.key)
               responses += OasResponseParser(
                 entry.value.as[YMap], { r =>
-                  r.set(ResponseModel.Name, node, Annotations(entry.key))
+                  r.withName(node)
```

Comment:
looks like lexical info is lost here by looking at oas30api.expanded.jsonld change

Reply:
The method with name sets the field as inferred (because key "name" is not present, only the value).
On the other hand, Value.value annotations (name value) should be present, right?